### ISSUE TITLE
for anyone who don't want to build by himself, here is a package to install

### ISSUE THREAD

**Issue by realysy** [m2-farzan/ros2-humble-PKGBUILD#8]
Look here: [https://github.com/realysy/ros_packages_for_arch_manjaro_linux](https://github.com/realysy/ros_packages_for_arch_manjaro_linux)

Thanks for the work from this repository~

**Comment by m2-farzan** [comment]
Hi

Thanks for sharing! I once attempted to publish a similar binary package but it didn't work on other systems where my build directory didn't exist. Did you test your package in systems other than the one you built it on?

Another point is the fact that the package may get broken when the dependencies update. So keeping it for long term is practically a debt, even if we don't aim to support versions other than 2022.11.23.

What are your thoughts on these?

**Comment by realysy** [comment]
> Hi
> 
> Thanks for sharing! I once attempted to publish a similar binary package but it didn't work on other systems where my build directory didn't exist. Did you test your package in systems other than the one you built it on?
> 
> Another point is the fact that the package may get broken when the dependencies update. So keeping it for long term is practically a debt, even if we don't aim to support versions other than 2022.11.23.
> 
> What are your thoughts on these?

Yes, I have installed the package in another system (a virtual machine) and tested the turtlesim demo, it worked as expected. 

As for the dependency conflicts that may occur in the futhre, I didn't think that far away.... I even want to create a package that contains the ROS package itself and all of its dependencies, so that I only need to extract them on a new machine.

**Comment by petronny** [comment]
Hi, the [arch4edu](https://github.com/arch4edu/arch4edu) repository now also provides the binaries for this package and all its dependencies. You can find the status and build logs for these packages at https://arch4edu.org/ under the `x86_64/ros2` catagory.

> Another point is the fact that the package may get broken when the dependencies update. So keeping it for long term is practically a debt, even if we don't aim to support versions other than 2022.11.23.

Arch4edu provides `arch4edu.links.db` for broken dynamic links detection. The automatic rebuild is possible but still under developement. Please feel free to discuss and contribute.
